geniesp/bpc_redcap_export_mapping.py

```diff
@@ -1771,7 +1790,7 @@ def get_sample(self, df_map: pd.DataFrame, df_file: pd.DataFrame) -> pd.DataFram
         df_sample_subset["AGE_AT_SEQUENCING"] = df_sample_subset[
             "AGE_AT_SEQUENCING"
         ].apply(math.floor)
-        # Remove SAMPLE_TYPE and CPT_SEQ_DATE because the values are incorrect
+        # Remove CPT_SEQ_DATE because the values are incorrect
```

Comment:
Check if we need to get SEQ_YEAR from main GENIE

Reply:
During table update, the cpt_seq_date has been overwritten by SEQ_YEAR from main GENIE. However, it should pull the data from the last consortium release that is tied to the most recent public release instead of pulling from the sample clinical database (as the code below).
Code: https://github.com/Sage-Bionetworks/genie-bpc-pipeline/blob/develop/scripts/table_updates/update_data_table.py#L228

Reply:
https://sagebionetworks.jira.com/browse/GEN-879